src/hooks/useAsset.js

```diff
@@ -58,28 +57,14 @@ export default function useAsset(asset) {
   const genericAssets = useSelector(
     ({ data: { genericAssets } }) => genericAssets
   );
-  const uniswapAssetsInWallet = useUniswapAssetsInWallet();
 
   return useMemo(() => {
     if (!asset) return null;
 
     let matched = null;
     if (asset.type === AssetTypes.token) {
-      const uniswapAsset = find(
-        uniswapAssetsInWallet,
-        matchesProperty('address', asset.mainnet_address || asset.address)
-      );
+      matched = find(allAssets, matchesProperty('uniqueId', asset.uniqueId));
```

Comment:
so now we never look by `address`? This is weird because I introduced matching by `uniqueId` a few days ago